### PR TITLE
Added information about using pg-index-health in unit tests.

### PR DESCRIPTION
**Pull request by mfvanek**
#65

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,12 +28,12 @@
 
 For raw sql queries see [pg-index-health-sql](https://github.com/mfvanek/pg-index-health-sql) project.
 
-## Important note
-**pg_index_health** uses the [PostgreSQL's statistics collector](https://www.postgresql.org/docs/10/monitoring-stats.html).  
-You can call `pg_stat_reset()` to reset all statistics counters for the current database to zero.
+## How does it work
+**pg_index_health** utilizes the [PostgreSQL's statistics collector](https://www.postgresql.org/docs/10/monitoring-stats.html).  
+You can call `pg_stat_reset()` on each host to reset all statistics counters for the current database to zero
+but the best way todo it is to use [IndexesHealth::resetStatistics()](https://github.com/mfvanek/pg-index-health/blob/9251f99e2952bc7490137f40c83873ff54ac1ffa/src/main/java/io/github/mfvanek/pg/index/health/IndexesHealth.java#L168) method.
```
Comment (kondratyev-nv) [inline README.md]: to do (add space, please)

File: README.md [hunk 2]
```diff
@@ -48,75 +48,17 @@ Using Maven:
...
+All these cases are covered with examples in the [pg-index-health-demo](https://github.com/mfvanek/pg-index-health-demo) project.
 
 ## Questions, issues, feature requests and contributions
+* If you have any question or a problem with the library, please [file an issue](https://github.com/mfvanek/pg-index-health/issues).
+* Contributions are always welcome! Please see [contributing guide](CONTRIBUTING.md) for more details.
 
- * If you have any question or a problem with the library, please [file an issue](https://github.com/mfvanek/pg-index-health/issues).
- * Contributions are always welcome! Please see [contributing guide](CONTRIBUTING.md) for more details.
+## Related projects
+[pg_analyse](https://github.com/idlesign/pg_analyse)
```
Comment (kondratyev-nv) [inline README.md]: I think it would be better to add some information about the project.

File: README.md (hunk 1 above)
Comment (mfvanek) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (mfvanek) [reply README.md]: fixed

**Pull request merged**